src/kirby/components/list/list.component.html

```diff
@@ -33,6 +34,7 @@
     <ion-item *ngFor="let item of items;trackBy: trackByFn"
               [class.has-divider]="showDivider"
               [class.selectable]="isSelectable"
+              [class.selected]="markSelectedRow && selectedItem && item.id === selectedItem.id"
```

Comment:
Same as other comment regarding the use of the `id` property.

Reply:
Using dynamic internal _id guid